utt/wallet-cli/src/main.cpp

```diff
@@ -54,11 +54,13 @@ void printHelp() {
   std::cout << "config                    -- configures wallets with the privacy application.\n";
   std::cout << "show                      -- prints information about the user managed by this wallet\n";
   std::cout << "register <user-id>        -- requests user registration required for spending coins\n";
-  std::cout << "mint <amount>             -- mint the requested amount of public funds.\n";
+  std::cout << "convertPublicToPrivate <amount>             -- converts the specified amount of public funds to "
+               "private funds.\n";
```

Comment:
(sorry for the annoying comment) 
lets be consistent. if you put a . at the end of the sentence then please add it to register also